### PR TITLE
refactor: upgrade template

### PR DESCRIPTION
**Pull request by kleyow**
Is it fine if I use, if not what's the licensing I should put in each file.
```
/**************************************************************************
 *  (C) Copyright Mojaloop Foundation 2020 - All rights reserved.         *
 *                                                                        *
 *  This file is made available under the terms of the license agreement  *
 *  specified in the corresponding source code repository.                *
 *                                                                        *
 *  Original Author:                                                      *
 *       Name - Email                                                     *
 *  Contributors:                                                         *
 *        Name - Email                                                    *
 **************************************************************************/
 ```
 
 A lot of the license/contributor are files pulled from other repos, I'll fix that in an upcoming commit once I know what to put for licensing.
 
 @vgenev Hi, hope you ok with these changes, trying to bring this up to date before we start building off this to make a oracle.
 @lewisdaly Ping on the reminder to make this a template in the settings.

### CODE REVIEW

**Comment by lewisdaly** [comment]
> @lewisdaly Ping on the reminder to make this a template in the settings.
Done

File: src/interface/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,809 @@
+openapi: 3.0.0
```
Comment (lewisdaly) [inline src/interface/openapi.yaml:1]: Does it make sense to use the api-snippets here?

**Review by elnyry-sam-k — DISMISSED** [review]

File: src/interface/openapi.yaml (hunk 1 above)
Comment (kleyow) [reply src/interface/openapi.yaml:1]: Yes it does. Will do that in this ticket.

Comment (lewisdaly) [reply src/interface/openapi.yaml:1]: Cool

**Review by sridharvoruganti — APPROVED** [review]
lgtm

**Pull request merged**